ext/libclementine-tagreader/tagreader.cpp

```diff
@@ -783,20 +787,20 @@ bool TagReader::SaveFile(const QString& filename,
   fileref->tag()->setAlbum(StdStringToTaglibString(song.album()));
   fileref->tag()->setGenre(StdStringToTaglibString(song.genre()));
   fileref->tag()->setComment(StdStringToTaglibString(song.comment()));
-  fileref->tag()->setYear(song.year());
-  fileref->tag()->setTrack(song.track());
+  fileref->tag()->setYear(song.year() <= 0 - 1 ? 0 : song.year());
```

Comment:
I'm not sure what the point of the `0 - 1` is?

Reply:
Matching other instances of the representation of '-1'.

Reply:
Fair enough.